### PR TITLE
Transfer service ecs task

### PR DESCRIPTION
**Pull request by TomJKing**
Will replace with `da-terraform-modules` when available.

### CODE REVIEW

**Review by Tom-Hallett — DISMISSED** [review]
All looks fine

File: root_transfer_service.tf [hunk 1]
```diff
@@ -42,3 +43,106 @@ module "transfer_service_task_policy" {
...
+  container_name               = "transfer-service"
+  cpu                          = 512
+  environment                  = local.environment
+  execution_role               = module.transfer_service_execution_role[0].role_arn
+  load_balancer_container_port = 8080
+  memory                       = 1024
+  private_subnets              = module.shared_vpc.private_backend_checks_subnets
+  security_groups              = [module.transfer_service_ecs_security_group[0].security_group_id]
+  service_name                 = "transfer_service_${local.environment}"
+  task_family_name             = "transfer_service-${local.environment}"
```
Comment (annielh) [inline root_transfer_service.tf:146]: Nit- choose `_` or `-` for both

**Review by annielh — APPROVED** [review]
All looks fine, tiny tiny comment.

I guess my only question would be how much work would it be to take the opportunity to add the necessary modules to `da-terraform-modules` now? Do we have a particular point where we're expecting them to become available through our or other teams work if we don't add them now / do we have a placeholder for the work?

File: root_transfer_service.tf (hunk 1 above)
Comment (TomJKing) [reply root_transfer_service.tf]: Good spot

**Comment by TomJKing** [comment]
> _I guess my only question would be how much work would it be to take the opportunity to add the necessary modules to da-terraform-modules now?_
@annielh I will need to add the new modules, I've started this work, but it will take a while for me to do it all. I want to get this ECS task stood up for a longer period of time to test stuff out in preparation to the SharePoint work. Which is why I'm using the tdr modules for now. Does that make sense?

**Review by annielh — APPROVED** [review]

**Pull request merged**